kubeflow/chainer-job/chainer-job.libsonnet

```diff
@@ -0,0 +1,127 @@
+local k = import "k.libsonnet";
```

Comment:
We are moving in the direction of putting the entire manifest in the jsonnet prototype as opposed to splitting it between a libsonnet and jsonnet file.See
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/examples/prototypes/tf-job-simple-v1alpha1.jsonnet

This way the result of ks generate is a jsonnet file that contains the complete manifest that users can easily modify. 
(i.e ks generate is just fancy copy and paste).

We are moving away from using complex jsonnet features to parameterize templates in complex ways because this hurts readability.

Reply:
Thanks for letting me know the direction!  I will change it 👍